tests/trace/test_actions_lifecycle.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+
+import weave
+from tests.trace.util import client_is_sqlite
+from weave.trace.weave_client import WeaveClient
+from weave.trace_server.interface.base_object_classes.actions import (
+    ActionDefinition,
+)
+from weave.trace_server.trace_server_interface import (
+    ActionsExecuteBatchReq,
+    FeedbackCreateReq,
+)
+
+
+def test_action_lifecycle_word_count(client: WeaveClient):
```

Comment:
What are the significant differences between this test and the one above?

I can see (but dont understand the significance of):
1. Difference in `ActionDefinition` spec
2. An added manual feedback (?)

Reply:
the word count action spec is stuper simple whereas the llm one requires the full litellm code path.

The manual part is just a demonstration of being able to manually create this feedback type - really only need to validate this once. Could be pulled out to it's own pr

Reply:
ok maybe this comment in the code could be helpful for others to understand the difference as well

Reply:
ack - adding now